src/screens/Game/index.tsx

```diff
@@ -33,7 +35,21 @@ const Game = () => {
   //   // :^)
   // }, []);
 
-  const handleSubmit = () => {
+  const isWord = async (word: string) => {
+    return axios
+      .get(
+        `${MW_API_URL}/api/v3/references/collegiate/json/${word}?key=${MW_API_KEY}`,
+      )
+      .then(res => {
```

Comment:
consider destructuring res and pulling data out of res. Might not be a bad idea to set up a type/model for the returned data structure in case the API ever changes.